### PR TITLE
[fix bug 1384843] Don't collect metrics if user have opted out from telemetry

### PR DESCRIPTION
**Pull request by glogiotatidis**
See also https://github.com/mozilla/activity-stream/commit/aeee212d3f0c4ede4647e065af8a328ca6f5bc11

### CODE REVIEW

**Comment by glogiotatidis** [comment]
@ScottDowne could you please r?

File: snippets/base/templates/base/includes/snippet_as.js [hunk 1]
```diff
@@ -421,7 +421,9 @@ function sendMetric(metric, callback, href) {
       }
       return;
     {% else %}
-      if (!SNIPPET_METRICS_URL || (Math.random() > SNIPPET_METRICS_SAMPLE_RATE) || (!metric)) {
+
+    if (gSnippetsMap.get('telemetryEnabled') !== true || !SNIPPET_METRICS_URL ||
```
Comment (ScottDowne) [inline snippets/base/templates/base/includes/snippet_as.js]: Any particular reason you used `gSnippetsMap.get('telemetryEnabled') !== true` and not `!gSnippetsMap.get('telemetryEnabled')`?

I'm fine either way, this is pretty simple.

**Comment by ScottDowne** [comment]
Just one small question because I was curious.

I can give it an r+ either way :)

File: snippets/base/templates/base/includes/snippet_as.js (hunk 1 above)
Comment (k88hudson) [inline snippets/base/templates/base/includes/snippet_as.js]: This should actually be `gSnippetsMap.get("appData.telemetryEnabled")` https://github.com/mozilla/activity-stream/blob/master/docs/v2-system-addon/snippets.md#expected-values-in-gsnippetsmap

Comment (glogiotatidis) [inline snippets/base/templates/base/includes/snippet_as.js]: > Any particular reason you used

Not really, changed to `!gSnippetsMap.get('appData.telemetryEnabled')`

**Comment by glogiotatidis** [comment]
Thanks both!

**Pull request merged**